app/models/recipe.rb

```diff
@@ -16,6 +17,10 @@ class Recipe < ActiveRecord::Base
 
   acts_as_taggable
 
+  def forked_from
+    Recipe.where(id: forked_from_recipe_id).first
```

Comment:
Oops I'm not sure how I ended up with that, I meant to say use the `find_by` method. It's like where but will return the first element.